test/e2e/app-dir/server-source-maps/server-source-maps.test.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path'
+import { nextTestSetup } from 'e2e-utils'
+
+describe('app-dir - server source maps', () => {
+  const { skipped, next, isNextDev } = nextTestSetup({
+    files: path.join(__dirname, 'fixtures/default'),
+    // Deploy tests don't have access to runtime logs.
+    // Manually verify that the runtime logs match.
+    skipDeployment: true,
```

Comment:
Manually verified the logs match a local `next-start`: https://vercel.com/vtest314-next-e2e-tests/vtest314-e2e-tests/833nSnKnDeLkcGUNgNRxfWP2nKzW/logs?slug=app-future&slug=en-US&slug=vtest314-next-e2e-tests&slug=vtest314-e2e-tests&slug=833nSnKnDeLkcGUNgNRxfWP2nKzW&slug=logs&page=1&timeline=past30Minutes&startDate=1729759798740&endDate=1729761598740&selectedLogId=1729761435556143555671100000&selectedLogTimestamp=1729761435556&forceEndDate=1729761435557